.github/workflows/test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node: [16.13.0, 18]
+        node: [16.13.0, 20]
```

Comment:
With Node 16 going EOL today, do we still want to build for it?

Reply:
However, could we bump this to 16.14 to be inline with Wrangler?